python/ray/data/tests/test_image.py

```diff
@@ -168,6 +168,55 @@ def get_relative_path(path: str) -> str:
             "image-datasets/simple/image3.jpg",
         ]
 
+    def test_random_shuffle(self, ray_start_regular_shared):
+        # NOTE: set preserve_order to True to allow consistent output behavior.
+        context = ray.data.DataContext.get_current()
+        preserve_order = context.execution_options.preserve_order
```

Comment:
nit, define a context manager in `conftest.py` for enabling preserve_order.  That would be clearer and reusable.

Reply:
We also have the restore_data_context fixture.

Reply:
+1 let's use a pytest fixture here